src/main/java/core/basesyntax/Lottery.java

```diff
@@ -0,0 +1,12 @@
+package core.basesyntax;
+
+import java.util.Random;
+
+public class Lottery {
+    public Ball getRandomBall() {
+        ColorSupplier colorSupplier = new ColorSupplier();
+        int index = new Random().nextInt(100);
+        Ball ball = new Ball(index, colorSupplier.getRandomColor());
```

Comment:
Let's move `ColorSupplier` and `Random` to class-level declaration.

Also, move the magic number `100` to constant.